packages/client/src/lobby/createReplayJSON.ts

```diff
@@ -207,7 +207,8 @@ function getCardFromHypoState(
   if (globals.state.replay.hypothetical === null) {
     return 0;
   }
-  return globals.state.replay.hypothetical.states[stateIndex].hands[
+  const minStateIndex = Math.max(stateIndex, 0);
+  return globals.state.replay.hypothetical.states[minStateIndex].hands[
```

Comment:
we can do better than this.
the line is still doing more than one thing.
keep breaking it up with good variable names that describe what is actually happening